fog/api/proto/report.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
```

Comment:
What is the workflow for updating this proto both for internal and mobilecoin?

Reply:
It's going to get removed from `internal` once this gets merged. If you're doing work that requires updating it, you'll need to first update it on this repo, or have 2 PRs going in parallel (and temporarily move your `internal/public` to point at your PR branch).

Reply:
Or is this where it lives now?

Reply:
Awesome - got it 👍